vsts/config/karma/test.karma.conf.js

```diff
@@ -13,7 +13,7 @@ const args = minimist(process.argv.slice(2));
  * @param {Object} config
  */
 function getConfig(config) {
-  shared(config, {
+  shared('vsts', config, {
```

Comment:
This should probably be `ado` instead of `vsts`.

Reply:
I agree, but we're using "vsts" everywhere else. The pipeline runs `skyux e2e --platform vsts` for example.

Reply:
I can change it here if you want, but it might cause confusion.